### ISSUE TITLE
 MOTA is higher by increasing the value of self.max_time_lost on MOT16-train

### ISSUE THREAD

**Issue by gmt710** [ifzhang/FairMOT#183]
I modify the value of self.max_time_lost, and get higher MOTA on MOT16-train:
before：
![image](https://user-images.githubusercontent.com/34018741/86782385-915fff80-c091-11ea-8185-91be27b384eb.png)

after：
![image](https://user-images.githubusercontent.com/34018741/86782361-8c02b500-c091-11ea-8afe-16ce46721087.png)

**Comment by NaifahNurya** [comment]
But when you look at the number of ID switch (IDS) is almost doubled. @gmt710

**Comment by gmt710** [comment]
@NaifahNurya Yes, I noticed it yet.Maybe this parameter can be adjusted again.
